internal/utils/docker.go

```diff
@@ -113,6 +113,11 @@ func DockerRemoveAll() {
 	wg.Wait()
 }
 
+func DockerRemoveAll(ctx context.Context, netId string) error {
+	DockerRemoveContainers(ctx)
+	return Docker.NetworkRemove(ctx, netId)
+}
```

Comment:
No need to return the error if we are ignoring it everywhere

```suggestion
func DockerRemoveAll(ctx context.Context, netId string) {
	DockerRemoveContainers(ctx)
	_ = Docker.NetworkRemove(ctx, netId)
}
```

Reply:
Makes, sense I wanted to return in to handle them in different PRs but is such a thing that is worth doing all at once (returning error and handling all their usage).

TL;DR will remove the return and open a new PR to handle them